src/test/java/edu/ksu/canvas/attendance/services/AttendanceSummaryCSVServiceUTest.java

```diff
@@ -72,9 +72,9 @@ public void createCsvSimpleAttendanceHappyPath() throws IOException {
         boolean isSimpleAttendance = true;
 
         StringBuilder shouldGetThis = new StringBuilder();
-        shouldGetThis.append("\"Name\",\"Total Classes Present\",\"Total Classes Absent\",\"Total Classes Tardy\",\"Total Classes Excused\"\n");
-        shouldGetThis.append("\""+ARBITRARY_STUDENT_NAME_1 +"\",\""+ARBITRARY_STUDENT_PRESENT_1+"\",\""+ARBITRARY_STUDENT_MISSED_1+"\",\""+ARBITRARY_STUDENT_TARDY_1+"\",\""+ ARBITRARY_STUDENT_EXCUSED_1 +"\"\n");
-        shouldGetThis.append("\""+ARBITRARY_STUDENT_NAME_2 +"\",\""+ARBITRARY_STUDENT_PRESENT_2+"\",\""+ARBITRARY_STUDENT_MISSED_2+"\",\""+ARBITRARY_STUDENT_TARDY_2+"\",\""+ ARBITRARY_STUDENT_EXCUSED_2 +"\"\n");
+        shouldGetThis.append("\"Name\",\"Total Classes Present\",\"Total Classes Tardy\",\"Total Classes Absent\",\"Total Classes Excused\"\n");
+        shouldGetThis.append("\""+ARBITRARY_STUDENT_NAME_1 +"\",\""+ARBITRARY_STUDENT_PRESENT_1+"\",\""+ARBITRARY_STUDENT_TARDY_1+"\",\""+ARBITRARY_STUDENT_MISSED_1+"\",\""+ ARBITRARY_STUDENT_EXCUSED_1 +"\"\n");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "\",\"" 16 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)